Library/Homebrew/cask/utils/copy-xattrs.swift

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/swift
+
+import Foundation
+
+struct SwiftErr: TextOutputStream {
+    public static var stream = SwiftErr()
+
+    mutating func write(_ string: String) {
+        fputs(string, stderr)
+    }
+}
+
+guard CommandLine.arguments.count >= 3 else {
+    print("Usage: swift copy-xattrs.swift <source> <dest>")
+    exit(2)
+}
+
+CommandLine.arguments[2].withCString { destPath in
+    let destNamesLen = listxattr(destPath, nil, 0, 0)
```

Comment:
```suggestion
    let destinationNamesLength = listxattr(destPath, nil, 0, 0)
```
and similar elsewhere